lib/ex_plasma.ex

```diff
@@ -10,13 +10,25 @@ defmodule ExPlasma do
   # when we introduce a new payment type, you name it `paymentV2`
   # https://github.com/omisego/plasma-contracts/blob/6ab35256b805e25cfc30d85f95f0616415220b20/plasma_framework/docs/design/tx-types-dependencies.md
   @payment_v1 <<1>>
+  @fee_token_claim_v1 <<3>>
```

Comment:
I don't think we need the version here yet. Additionally spoke with the other team and I think we can just simplify this name to `fee`